src/main/resources/templates/db_scripts/schema.json

```diff
@@ -401,13 +401,13 @@
         },
         {
           "fieldName": "indexTitle",
-          "tOps": "ADD",
+          "tOps": "DELETE",
           "caseSensitive": false,
           "removeAccents": true
         },
         {
           "fieldName": "title",
-          "tOps": "ADD",
+          "tOps": "DELETE",
```

Comment:
Can we keep these 2 indexes? The title index is used by FQM and the indexTitle index probably should be, so removing them would hurt FQM's performance

Reply:
A list made with the query `instance.title == 'abc'` would use that index. And I can't remember for sure, but I believe `instance.title starts_with 'abc'` would, too (I can't remember for sure on that one, though... We still might be waiting on the changes in FQM that would make that the case)

Reply:
It's worth noting that we work to set up fields in FQM to use indexes wherever possible. So even if the index is `"left"(lower(diku_mod_inventory_storage.f_unaccent(jsonb ->> 'title')), 600)` or something like that, FQM can use it.

Reply:
@julianladisch, @mweaver-ebsco. Is there any agreement achieved?

Reply:
Hey @julianladisch, seems that @mweaver-ebsco has just doubts but not strong objections. So, I'm approving the PR.